doc/code/overview.rst

```diff
@@ -0,0 +1,91 @@
+.. _library_overview:
+
+Overview
+========
+
+.. currentmodule:: pennylane
+
+The code documentation serves as a reference where all classes and functions of
+PennyLane can be looked up. This is useful in order to check the signatures of inputs
+and outputs of functions, or the attributes and methods of classes.
+
+Functions and classes are organised in **modules**. To import a specific class or function, you need
+to specify the full module path, i.e.,
+
+.. code::
+
+    from pennylane.templates.layers import StronglyEntanglingLayer
+
+While some modules are relevant to users, others are providing lower-level functionality and are therefore
```

Comment:
While some modules are relevant to users, others provide lower-level functionality and are therefore

Reply:
Thanks Aroosa! In future, if you use the "insert a suggestion" button you can just code up your suggestion and I can commit it with the click of a button :)

Reply:
```suggestion
While some modules are relevant to users, others provide lower-level functionality and are therefore
```